packages/fast-form-generator-react/src/class-name-contracts/form-item.select.class-name-contract.ts

```diff
@@ -3,7 +3,10 @@
  */
 export default interface FormItemSelectClassNameContract {
     formItemSelect: string;
-    formItemSelect_label: string;
-    formItemSelect_span: string;
-    formItemSelect_input: string;
+    formItemSelect_control: string;
+    formItemSelect_control_label: string;
+    formItemSelect_control_span: string;
+    formItemSelect_control_input: string;
+    formItemSelect_softRemove: string;
```

Comment:
What is a "softRemove"

Reply:
It will remove the data, but its a soft remove so if you click it you will add back in the data you just removed.